examples/configurations/api/anthropic_api.py

```diff
@@ -0,0 +1,19 @@
+import os
+from llama_index.llms.anthropic import Anthropic
+
+
+class AnthropicApiLLM(Anthropic):
+    # This class initializes the Anthropic API with selected models.
+    # Available models: claude-3-haiku-20240307, claude-3-sonnet-20240229, claude-3-opus-20240229
+    def __init__(
+            self,
+            model: str = "claude-3-haiku-20240307",
+            api_key: str = os.getenv("ANTHROPIC_API_KEY"),
+    ):
+        if api_key is None:
+            raise ValueError("ANTHROPIC_API_KEY environment variable is not set")
+
+        super().__init__(
+            model=model,
+            api_key=api_key,
+        )
```

Comment:
We will also need to add the anthropic module to the pyproject.toml file - I think it will need to be "llama-index-llms-anthropic==0.1.8" for compatibility reasons with LaVague